StripePaymentSheet/StripePaymentSheet/Source/Helpers/PaymentSheetLinkAccount.swift

```diff
@@ -428,7 +429,7 @@ extension PaymentSheetLinkAccount {
         params.link?.paymentDetailsID = paymentDetails.stripeID
         params.link?.credentials = ["consumer_session_client_secret": currentSession.clientSecret]
 
-        if let cvc = paymentDetails.cvc {
+        if let cvc = cvc {
```

Comment:
I'm a little confused.  Previously, we were using ConsumerPaymentDetails.Details to pass around a CVC, but now we are passing it in directly?

Should we remove `ConsumerPaymentDetails.cvc`, or additional code in ConsumerPaymentDetails so that we don't use it?

Reply:
Yeah, this is unused code from Link v1. I'll rip it out.